main/guides/getting-started/syncing-up.md

```diff
@@ -0,0 +1,86 @@
+# Syncing Up on Mainnet
+
+To run a node on the `agoric-3` network from scratch, please follow the 
+steps below.
+
+If you wish to restore from a recent community snapshot, please follow the instructions from the heading [Current Community Snapshot](#current-community-snapshot).
```

Comment:
We should probably let folks know that if they choose not to use a community snapshot, they should expect to spend a few days catching up.